totalRP3_Extended/aura/aura.xml

```diff
@@ -0,0 +1,244 @@
+		<Size x="200" y="50"/>
+		<Anchors>
+			<Anchor point="BOTTOMRIGHT" x="-50" y="0" relativePoint="BOTTOMLEFT" relativeTo="Minimap" />
+		</Anchors>
+		<Layers>
+			<Layer level="ARTWORK">
+				<FontString parentKey="Text" text="" inherits="GameFontNormal" justifyH="CENTER" justifyV="BOTTOM" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT"/>
+						<Anchor point="BOTTOMRIGHT" relativePoint="TOPRIGHT" x="0" y="-16"/>
+					</Anchors>
+					<Color r="0.9" g="0.9" b="0.9"/>
+				</FontString>
+			</Layer>
+			<Layer level="BACKGROUND">
+				<Texture parentKey="BottomLeft" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="BOTTOMLEFT"/>
+					</Anchors>
+					<TexCoords top="0.03125" left="0.015625" bottom="0.53125" right="0.265625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="BottomRight" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="BOTTOMRIGHT"/>
+					</Anchors>
+					<TexCoords top="0.03125" left="0.265625" bottom="0.53125" right="0.015625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopLeft" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="TOPLEFT"/>
+					</Anchors>
+					<TexCoords top="0.53125" left="0.015625" bottom="0.03125" right="0.265625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopRight" file="Interface\FrameGeneral\UIFrameBackground" hidden="true">
+					<Size x="16" y="16"/>
+					<Anchors>
+						<Anchor point="TOPRIGHT"/>
+					</Anchors>
+					<TexCoords top="0.53125" left="0.265625" bottom="0.03125" right="0.015625"/>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="TopEdge" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.TopLeft" relativePoint="TOPRIGHT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.TopRight" relativePoint="BOTTOMLEFT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="BottomEdge" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.BottomLeft" relativePoint="TOPRIGHT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.BottomRight" relativePoint="BOTTOMLEFT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
+				<Texture parentKey="Center" hidden="true">
+					<Anchors>
+						<Anchor point="TOPLEFT" relativeKey="$parent.TopLeft" relativePoint="BOTTOMLEFT"/>
+						<Anchor point="BOTTOMRIGHT" relativeKey="$parent.BottomRight" relativePoint="TOPRIGHT"/>
+					</Anchors>
+					<Color r="0.12" g="0.12" b="0.12" a="0.5"/>
+				</Texture>
```

Comment:
This whole section _may_ be removable if you instead inherit from FlatPanelBackgroundTemplate. It might look a bit different than you've intended though.

Reply:
I left the display as is for now and let others decide if the design is okay (still fixed the code style comments).
Purpose is to give a clear indication to the user what those icons are when they hover them and at the same time show as little as possible when they don't.

Another possiblity is separate buff and debuff bars.